uitools/examples/src/BaseDemo.cpp

```diff
@@ -22,7 +23,6 @@
 #include <MapViewTypes.h>
 #include <Scene.h>
 #include <SceneQuickView.h>
-#include <stdexcept>
```

Comment:
I couldn't find anything needing this. I didn't go bananas pruning includes, but this one seemed as safe as it gets.